File: common/src/main/java/generations/gg/generations/core/generationscore/world/level/block/entities/generic/GenericModelProvidingBlockEntity.java
package generations.gg.generations.core.generationscore.world.level.block.entities.generic;

import generations.gg.generations.core.generationscore.world.level.block.entities.ModelProvidingBlockEntity;
import generations.gg.generations.core.generationscore.world.level.block.entities.GenerationsBlockEntities;
import net.minecraft.core.BlockPos;
import net.minecraft.world.level.block.state.BlockState;

public class GenericModelProvidingBlockEntity extends ModelProvidingBlockEntity {
    public GenericModelProvidingBlockEntity(BlockPos arg2, BlockState arg3) {
        super(GenerationsBlockEntities.GENERIC_MODEL_PROVIDING.get(), arg2, arg3);
    }
}

File: common/src/main/java/generations/gg/generations/core/generationscore/common/world/level/block/entities/generic/GenericModelProvidingBlockEntity.kt
package generations.gg.generations.core.generationscore.common.world.level.block.entities.generic

import generations.gg.generations.core.generationscore.common.world.level.block.entities.GenerationsBlockEntities
import generations.gg.generations.core.generationscore.common.world.level.block.entities.ModelProvidingBlockEntity
import net.minecraft.core.BlockPos
import net.minecraft.world.level.block.state.BlockState

class GenericModelProvidingBlockEntity(pos: BlockPos, state: BlockState) : ModelProvidingBlockEntity(GenerationsBlockEntities.GENERIC_MODEL_PROVIDING, pos, state)
